### ISSUE TITLE
missing button mnemonics in Find and Replace Column Values window

### ISSUE THREAD

**Issue by cseppan** [USEPA-OAQPS/emf#82]
Data Editor, click button "Find and Replace Column Values" toolbar button
Add mnemonics to Apply (A) and Close (L) buttons in window
